### PR TITLE
Kevin/paper handler cli set scan dir

### PR DESCRIPTION
**Pull request by kshen0**
## Overview

* Adds support for `setScanDirection` to the driver CLI for manual testing.
* Moves the call to set default scan direction out of `driver.scanAndSave` and into the application layer. Always setting scan direction to `backward` in `scanAndSave` doesn't make a lot of sense because any previous call to `setScanDirection` would be overwritten right before the scan actually happens.
* Fixes a typo where the `in_park` scan direction was encoded with `0x02` instead of `0x03`

## Demo Video or Screenshot

N/A

## Testing Plan

* Ran CLI and confirmed the setting works

## Checklist

- [ ] (No new actions) I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

### CODE REVIEW

File: apps/mark-scan/backend/src/custom-paper-handler/application_driver.ts [hunk 1]
```diff
@@ -36,6 +37,10 @@ export async function setDefaults(
   debug('set line spacing to 0');
   await driver.setPrintingSpeed('slow');
   debug('set printing speed to slow');
+
+  const scanDirection: ScanDirection = 'backward';
+  await driver.setScanDirection(scanDirection);
+  debug('set scan direction to', scanDirection);
```
Comment (kshen0) [inline apps/mark-scan/backend/src/custom-paper-handler/application_driver.ts:43]: For https://github.com/votingworks/vxsuite/issues/4812 you may need to move `setScanDirection` into individual states, but you could also leave this default, change the direction for the re-cast flow, and change it back after. Up to you.

File: libs/custom-paper-handler/src/driver/scanner_config.ts [hunk 2]
```diff
@@ -9,7 +9,8 @@ export type PaperMovementAfterScan =
 export type ScanLight = 'red' | 'green' | 'blue' | 'white';
 export type ScanDataFormat = 'BW' | 'grayscale';
 export type Resolution = 100 | 150 | 200 | 250 | 300;
-export type ScanDirection = 'forward' | 'backward' | 'in_park';
+export const scanDirections = ['forward', 'backward', 'in_park'] as const;
```
Comment (kshen0) [inline libs/custom-paper-handler/src/driver/scanner_config.ts:12]: Allows printing of all valid scan directions in the CLI here: https://github.com/votingworks/vxsuite/pull/4993/files#diff-9b400756f5e8f9849559bce70d1e273e357200c02fea2d5e808e58e33f702ad4R133

File: libs/custom-paper-handler/src/driver/coders.ts [hunk 3]
```diff
@@ -207,7 +207,7 @@ export enum ConfigureScannerOptionSensorConfig {
 export enum ConfigureScannerFlags {
   NONE = 0x00,
   SCAN_BACKWARDS = 0x01,
-  SCAN_IN_PARK = 0x02,
+  SCAN_IN_PARK = 0x03,
```
Comment (kofi-q) [inline libs/custom-paper-handler/src/driver/coders.ts:210]: For my understanding: this is essentially setting both the `0x01` and `0x02` flags, right?

**Review by kofi-q — APPROVED** [review]
🎉

File: libs/custom-paper-handler/src/driver/scanner_config.ts [hunk 4]
```diff
@@ -9,7 +9,8 @@ export type PaperMovementAfterScan =
...
+export type ScanDirection = (typeof scanDirections)[number];
```
Comment (kofi-q) [inline libs/custom-paper-handler/src/driver/scanner_config.ts:13]: nit: Actually, could you also add a jsdoc comment to `ScanDirection` describing the behaviour of each direction setting?

File: libs/custom-paper-handler/src/driver/coders.ts (hunk 3 above)
Comment (kshen0) [reply libs/custom-paper-handler/src/driver/coders.ts:210]: Correct!

**Pull request merged**